package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "raspiblitz-web",
-  "version": "0.1.0",
+  "config": {
+    "port": "3000"
+  },
+  "version": "0.1.1",
```

Comment:
@cstenglein let me know when you want to talk about automatic versioning ;)

Reply:
that was from a test - putting back to 0.1.0